rest/route/host_create_test.go

```diff
@@ -79,7 +79,9 @@ func TestMakeHost(t *testing.T) {
 	}
 	handler.createHost = c
 	handler.taskID = "task-id"
-	h, err := data.MakeHost(ctx, env, handler.taskID, "", "", handler.createHost)
+	foundDistro, err := data.GetHostCreateDistro(ctx, c)
+	assert.NoError(err)
+	h, err := data.MakeHost(ctx, env, handler.taskID, "", "", handler.createHost, *foundDistro)
```

Comment:
This should either `require.NotZero(foundDistro)` or `require.NoError(err)` since L84 can panic if `foundDistro` is a nil pointer (same with the others in the file).